src/ripple/app/main/Main.cpp

```diff
@@ -360,6 +347,23 @@ int run (int argc, char** argv)
         config->START_UP = Config::LOAD;
     }
 
+    if (vm.count ("standalone"))
+    {
+        config->RUN_STANDALONE = true;
+        config->LEDGER_HISTORY = 0;
+        if (config->START_UP != Config::LOAD &&
+                config->START_UP != Config::LOAD_FILE &&
+                    config->START_UP != Config::REPLAY)
+            config->legacy("database_path", "");
```

Comment:
> Logging a warning that the configured value is being overridden would be nice. 

That's a good idea. There are two problems. 1) It'll have to be to `std::cerr`, because I don't think the logger is available at this point. 2) `Config::setup` will set `database_path` to a value regardless.

> Note this logic seems already in place:

The `DatabaseCon` check is redundant, but not global. There are several other consumers of `database_path`. I changed the value here in main to keep everything consistent, but left the check in `DatabaseCon`

> Is there any reason the updateTables() routines couldn't simply always be ran ?

Probably not. :)

After sleeping on this, and reading your feedback, I'm starting to think this PR is turning into a hot mess. I'll see if I can recover it quickly, else I'll close it and try again later.